README.md

```diff
@@ -28,13 +27,298 @@ pip install -U pyinstaller-hooks-contrib
+You may skip the 2<sup>nd</sup> line if you have no intention of providing tests (but please do provide tests!).
+
+
+### Add the hook
+
+Hooks live in the [src/_pyinstaller_hooks_contrib/hooks/stdhooks/](../master/src/_pyinstaller_hooks_contrib/hooks/stdhooks/) directory.
+Simply copy your hook into there.
+Please annotate (with comments) anything unusual in the hook.
+*Unusual* here is defined as any of the following:
+
+*   Long lists of `hiddenimport` submodules.
+    If you really need lots of hidden imports then use [`collect_submodules('foo')`](https://pyinstaller.readthedocs.io/en/latest/hooks.html#PyInstaller.utils.hooks.collect_submodules).
+    For bonus points, track down why so many submodules are hidden. Typical causes are:
+    *   Lazily loaded submodules (`importlib.importmodule()` inside a module `__getattr__()`).
+    *   Dynamically loaded *backends*.
+    *   Usage of `Cython` or Python extension modules containing `import` statements.
+*   Use of [`collect_all()`](https://pyinstaller.readthedocs.io/en/latest/hooks.html#PyInstaller.utils.hooks.collect_all).
+    This function is extremely inefficient and is really only for emergencies.
+    Check that you really do need to collect all of submodules, data files, binaries, metadata and dependencies.
+    If you do then add a comment to say so (and if you know it - why).
+    Do not simply use `collect_all()` just to *future proof* the hook.
+*   Any complicated `os.path` arithmetic (by which I mean overly complex filename manipulations).
+
+
+#### Add the copyright header
+
+All source files must contain the copyright header to be covered by our terms and conditions.
+
+If you are **adding** a new hook (or any new python file), copy/paste the appropriate copyright header (below) at the top
+replacing 2021 with the current year.
+
+<details><summary>GPL 2 header for standard hooks or other Python files.</summary>
+
+```python
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+```
+
+</details>
+
+<details><summary>APL header for runtime hooks only.
+If you're unsure if your hook is a runtime hook then it almost certainly is a standard hook.</summary>
+
+```python
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
```

Comment:
You appear to have duplicated the headers here - this one should be Apache, I believe.

Reply:
Fixed.